packages/SwingSet/test/test-kernel.js

```diff
@@ -8,6 +8,8 @@ import buildKernel from '../src/kernel/index';
 import { makeVatSlot } from '../src/parseVatSlots';
 import { checkKT } from './util';
 
+const RETIRE_KPIDS = true;
```

Comment:
I like having a switch here. Would the test actually pass if it got flipped? I'm thinking the `refCount` checks this adds would have different values if we were to disable `RETIRE_KPIDS`. If that's too much of a hassle to make work, feel free to remove the switch, but maybe leave a comment next to the promise clauses that would have to be removed if we were to disable it.